h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_gzip_large.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import sys
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import random
+
+def import_gzip_skipped_columns():
```

Comment:
Why the "large" postfix in the filename ?

Reply:
It depends on the developer's body mass.  In my case, I am fat and therefore my tests are also large.